mappings/net/minecraft/block/Material.mapping

```diff
@@ -3,54 +3,76 @@ CLASS net/minecraft/class_3614 net/minecraft/block/Material
 	FIELD field_15912 breakByHand Z
 	FIELD field_15913 COBWEB Lnet/minecraft/class_3614;
 	FIELD field_15914 STONE Lnet/minecraft/class_3614;
+		COMMENT Hard solid nonmetallic mineral matter of which rock is made, especially as a building material.
 	FIELD field_15915 BUBBLE_COLUMN Lnet/minecraft/class_3614;
-	FIELD field_15916 SAND Lnet/minecraft/class_3614;
+	FIELD field_15916 AGGREGATE Lnet/minecraft/class_3614;
+		COMMENT A material or structure formed from a loosely compacted mass of fragments or particles.
 	FIELD field_15917 SPONGE Lnet/minecraft/class_3614;
 	FIELD field_15918 REDSTONE_LAMP Lnet/minecraft/class_3614;
 	FIELD field_15919 PORTAL Lnet/minecraft/class_3614;
+		COMMENT Material for the various portal blocks.
 	FIELD field_15920 WATER Lnet/minecraft/class_3614;
 	FIELD field_15921 UNUSED_PLANT Lnet/minecraft/class_3614;
+		COMMENT Not in use, but has foliage color.
 	FIELD field_15922 LAVA Lnet/minecraft/class_3614;
 	FIELD field_15923 LEAVES Lnet/minecraft/class_3614;
-	FIELD field_15924 PART Lnet/minecraft/class_3614;
+	FIELD field_15924 SUB_BLOCK Lnet/minecraft/class_3614;
```

Comment:
Sub block seems like a bit of an odd name to me because they're still just normal blocks. I think PART is just fine personally.

Reply:
"Part" to me seems like a part of something, and since MC doesn't have multiparts or anything really that is assembled from "parts" it seems odd to me.

Reply:
`PARTIAL_BLOCK`?

Reply:
How is "partial" any different from "sub"?

Reply:
sub usually indicates an inclusion, so people would assume it to belong to a bigger block or a multiblock. partial doesn't have this ambiguity

Reply:
CHanged to PARTIAL